lib/rexml/xpath_parser.rb

```diff
@@ -147,7 +147,7 @@ def match(path_stack, nodeset)
         unnode(result)
       else
         [result]
-      end
+      end.uniq
```

Comment:
Do we need `.uniq` for the `[result]` case?

Reply:
In the case of `[result]`, it is not necessary.
I would remove it.
Thanks!